task/02-numbers-tasks.js

```diff
@@ -221,7 +222,7 @@ function isPrime(n) {
  *   toNumber(new Number(42), 0) => 42
  */
 function toNumber(value, def) {
-    throw new Error('Not implemented');
+    if (Number.isNaN(Number.parseFloat(value))) { return def; } return Number.parseFloat(value);
```

Comment:
please use a newline for every logical step in the future, this is more readable